docs/source/compression/index.rst

```diff
@@ -140,8 +140,8 @@ The following figure shows how NNI prunes and speeds up your models.
    :scale: 40%
    :alt:
 
-The detailed tutorial of Speed Up Model with Mask can be found :doc:`here <../tutorials/pruning_speed_up>`.
-The detailed tutorial of Speed Up Model with Calibration Config can be found :doc:`here <../tutorials/quantization_speed_up>`.
+The detailed tutorial of Speedup Model with Mask can be found :doc:`here <../tutorials/pruning_speed_up>`.
```

Comment:
Rename tutorial?
I know it's painful though...

Reply:
A... I will rename them